src/main/java/com/google/sps/data/BasicGameData.java

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.lang.Math;
+
+/**
+* Represents only the info that is needed about a game from the main page.
+*/
+public class BasicGameData {
+    //TODO(ldchen): delete all of the mocking once datastore is working.
+    private int getRandomIntegerBetween(int l, int r) {
+        int res = l + (int)(Math.random()*(r-l+1));
+        if(res > r) res = r; // happens if Math.random() somehow gives exactly 1
+        return res;
+    }
+    private String getRandomWord() {
```

Comment:
add extra line here between last function and this function

Reply:
Done, thanks.